CONTRIBUTING.md

```diff
@@ -251,7 +251,7 @@ Below is a checklist of things to be mindful of when implementing a new instrume
 
 - Follow semantic conventions
   - The instrumentation should follow the semantic conventions defined [here](https://github.com/open-telemetry/semantic-conventions/tree/main/docs)
-  - To ensure consistency, we encourage contributions that align with the stable semantic conventions. This approach helps us avoid potential confusion and reduces the need to support multiple deprecated versions of semantic conventions. However, we are still open to considering exceptional cases where changes are well justified.
+  - To ensure quality and consistency, we seek to avoid supporting multiple outdated versions of HTTP semantic conventions. Contributions related to outdated HTTP semantic conventions will likely be discouraged, as they increase complexity and the potential for misconceptions. However, we are still open to considering exceptional cases where changes are well justified.
```

Comment:
You lost `we encourage contributions that align with the stable semantic conventions` though that seems important to me. Since this is a general rule and you are adding one specific for HTTP I'd keep the previous as is and add a new one talking specifically about HTTP semconv